### DISCUSSION TITLE
Saturn 2 Testing Volunteer

### DISCUSSION THREAD

**Discussion by DonChooluvit** [rudetrooper/Octoprint-Chituboard#36] [General]
@rudetrooper, if you are working on the plugin supporting the Saturn 2, I am willing to volunteer to assist. I have everything set up and confirmed the plugin can control the movement of the build plate.

**Comment by jshank** [comment]
I'm still trying to track down the hardware. What features don't work on the Saturn 2? Are you able to add files to the virtual drive but not start the print? Monitor progress?

**Reply by DonChooluvit** [reply]
Replied in the other thread, but it's easier to say what features do work. 

- Camera works using the rPi camera only. 
- You can move the print head up and down, and home it via the plugin. 
- You can upload files to the Virtual USB, but it cannot send them to the printer, nor can the printer see it. 
- Looking at the terminal output while it is printing, there is a massive fraction in the billionths (X,XXX,XXX,XXX/XX,XXX,XXX,XXX) and if you do the math, you can get the progress in a percentage. 

That's it for now. I still use it, and wait for any future functionality.

**Comment by jshank** [comment]
Interesting. "You can upload files to the Virtual USB, but it cannot send them to the printer, nor can the printer see it." is the dealbreaker for me. Strange that the emulated FAT32 partition isn't seen. I'm still hunting for a compatible Pi and then will give this a try and see if I can't help progress the code with some PRs. C'mon rpilocator.com.

**Comment by DonChooluvit** [comment]
@rudetrooper has there been any progress on working version for the encrypted files?
